src/lib/auth/strategies/wpcom/oauth.js

```diff
@@ -0,0 +1,199 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import wpComRequest from 'wpcom-xhr-request';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import { WPcomStrategy } from './index';
+
+/**
+ * Module variables
+ */
+const debug = debugFactory( 'happychat-client:auth:wpcom:oauth' );
+const oAuth = require( 'wpcom-oauth-cors' )( '54006' );
```

Comment:
What is the `54006` constant for here? Is this something that may ever change? Maybe we can move it to a config file instead?

Reply:
it was part of a config file but forgot to move it back. Will do 🙇